src/index.js

```diff
@@ -1,9 +1,10 @@
-import './locales'
+// import './locales'
```

Comment:
This line should probably be uncommented, but I'm not sure how to generate the `./locales` directory.

Reply:
@HendrikThePendric `yarn build` will generate the directory, it's part of the i18n extract/generate build steps

Reply:
Without this line there will be no translations

Reply:
Yup, I can confirm running `yarn build` did the trick. Uncommented this line in 3323118.